pkg/project/registry/rest.go

```diff
@@ -0,0 +1,68 @@
+package registry
```

Comment:
Our packages appear to be `pkg/<sub-component(bundles related resources)>/<api(types)|registry(storage)|controller>/<single-resource>`.  That seems like a pretty reasonable way to slice the package that would allow the strategy to be shared by all different storage mechanism for a single resource.  Having it in the root of `registry` means that for our other packages, we'd get 4+ implementations (`images` for example) all crammed into a single file.  I could see an argument for splitting the `etcd` package down into the `single-resource` level, but project doesn't have one.